src/sentry/testutils/cases.py

```diff
@@ -3469,8 +3469,8 @@ def populate_project1(self) -> None:
             )
         ]
 
-    def load_errors(self) -> tuple[Event, Event]:
-        """Generates 2 events for gen1 projects."""
+    def load_errors(self) -> tuple[Event, Event, Event]:
```

Comment:
nit: if we're extending this to this point should we just return a list of events instead of a tuple?

Reply:
Sure 👍🏻